docs/processors/conditions.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Conditional Execution'
```

Comment:
We should also mention that the go template can use any function provided by [sprig](https://masterminds.github.io/sprig/).